support/hololens/ServoApp/ServoControl/Servo.cpp

```diff
@@ -40,6 +40,23 @@ Servo::Servo(GLsizei width, GLsizei height, ServoDelegate &aDelegate)
   o.enable_subpixel_text_antialiasing = false;
   o.vr_pointer = NULL;
 
+  // 7 filter modules.
+  static char *pfilters[64] = {
```

Comment:
Let's leave this commented out by default, so it's easy to enable it for testing purposes.

Reply:
Commented out.  Implemented.

Reply:
We will need to set:
```
o.vslogger_mod_list = NULL;
o.vslogger_mod_size = 0;
```
When **capi::CInitOptions o;** is initialized in C/C++ code such as Hololens and NO modules are passed in.  Otherwise opts.vslogger_mod_list and opts.vslogger_mod_size will have an undertermined value that will lead to an Access Violation.

Reply:
Also, when they are set to NULL/0, there is no logging.  In otherwords:
```
impl log::Log for VSLogger {
    fn enabled(&self, metadata: &Metadata) -> bool {
        let is_module_enabled = LOG_MODULE_FILTERS
            .lock()
            .unwrap()
            .contains(&String::from(metadata.target()));
        return metadata.level() <= Level::Warn && is_module_enabled;
    }
```

Should it be changed such that if there is no modules to filter on, all logging is enabled?

Reply:
Ah, right. Let's do:
```rust
let modules = LOG_MODULE_FILTERS.lock().unwrap();
let is_module_enabled = modules.contains(metadata.target()) || modules.is_empty();
return metadata.level() <= Level::Warn && is_module_enabled;
```

Reply:
Working on it now.

Reply:
Implemented.

Reply:
Implemented and tested.